ebos/eclbaseaquifermodel.hh

```diff
@@ -64,6 +68,18 @@ public:
     void initialSolutionApplied()
     { }
 
+    /*!
+     * \brief Called if aquifers are being initialized from values retrieved
+     *        from a restart file.
+     *
+     * \param[in] aquiferSoln Set of aquifer-related initial values, mostly
+     *        pertaining to analytic aquifers.  Contains at minimum the
+     *        aquifer pressure and the base run's total produced liquid
+     *        volume from the model's aquifers.
+     */
+    void initFromRestart(const std::vector<data::AquiferData>& aquiferSoln OPM_UNUSED)
+    { }
```

Comment:
Maybe we should throw here too to let users know we do not do this yet.

Reply:
> Maybe we should throw here too to let users know we do not do this yet.

I'll update accordingly.